app/templates/racer/index.html

```diff
@@ -1,5 +1,10 @@
 {% extends "base.html" %}
 
+{% block head %}
+{{super()}}
+<link rel="stylesheet" type="text/css" href="{{url_for('static', filename='bootstrap-slider.min.css')}}">
```

Comment:
@johnwheffner I know this is a hot topic of recent debate, but for consistency I would suggest using a [CDN](https://cdnjs.com/libraries/bootstrap-slider).

Thoughts?